packages/core/src/index.ts

```diff
@@ -128,7 +127,8 @@ const metric_strategyMaxChannels = create_gauge(
 const PACKET_QUEUE_TIMEOUT_MILLISECONDS = 15000n
 
 interface NetOptions {
-  ip: string
+  ip?: string
```

Comment:
I was just removing all of this code in the configuration refactor PR.

Reply:
As agreed will be done in config PR.